CONTRIBUTING.md

```diff
@@ -1,77 +1,304 @@
-# Contributing to convert2rhel
+# Contributing to Convert2RHEL
 
-## Coding guidelines
-1. All python code must be python 2.6/2.7/3.6 compatible
-1. The code should follow linting from pylint
+The following is a set of guidelines for contributing to Convert2RHEL codebase, which are hosted in
+the [OAMG Organization](https://github.com/oamg) on GitHub. These are mostly guidelines, not rules.
 
-## Developing locally
+<div align="center">
+
```

Comment:
Yeah, about that. I saw GitHub already has some TOC implemented for markdown stuff, should we go ahead and use theirs instead of this manual TOC? https://github.com/oamg/convert2rhel/pull/372#discussion_r755224423